onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlDFT.h

```diff
@@ -241,15 +364,15 @@ class GpuDFTOperator : public WRL::Base<IMLOperatorKernel>
 
                 ORT_THROW_HR_IF(E_INVALIDARG, !featureOptions.Native16BitShaderOpsSupported);
```

Comment:
>ORT_THROW_HR_IF(E_INVALIDARG, !featureOptions.Native16BitShaderOpsSupported); [](http://example.com/codeflow?start=16&length=77)

IIRC, this is already checked at a higher level when assigning operator nodes depending on supported device type, right? So then this statement is not expected to fail. Then I'd add a comment. e.g.

// Native16 should be supported by this device, because we would have already fallen back to CPU earlier otherwise.

Reply:
Yeah, forgot that i added a the logic into ExecutionProvider.cpp. Removed.
![image](https://user-images.githubusercontent.com/6754002/234905029-24ed3fc1-9a7b-4644-8258-2451920cc170.png)